### DISCUSSION TITLE
R - split testing dan kevalidan simulasi

### DISCUSSION THREAD

**Discussion by Hertzspr** [teamalgoritma/community#18] [Q&A] [answered]
Diketahui, sebuah tim sepak bola ingin menentukan formasi mana yang lebih efektif. 
Maka dijalankan simulasi dalam 3 bentuk tim. 
A dengan sayap posisi WB.
B dengan sayap posisi CWB.
C dengan sayap posisi WM.

Apakah cukup melakukan 10 simulasi musim yang masing-masing musim terdiri dari 36 pertandingan?
Bagaimana membuktikan hipotesis bahwa misalnya B lebih efektif dari C?

**Comment by muhrisman** [comment, marked as answer]
Dalam melakukan simulasi untuk memperbesar tingkat kepercayaan hasil yang didapatkan, bapak dapat melakukan bootsraping. Teknik bootstraping akan mencoba "menggandakan" data dengan melihat pola dari data yang terlihat. Hasil dari sample yang dilakukan bootstraping ini dapat dijadikan referensi untuk melihat seberapa jauh perbedaan strategi yang diterapkan. Adapun referensinya bisa bapak lihat pada referensi berikut:

https://rpubs.com/muh_risman/AB_Testing_With_R

**Reply by Hertzspr** [reply]
Terima kasih arahannya, nanti saya telusuri.

**Reply by Hertzspr** [reply]
Untuk mengecek p-value di rumusnya, kenapa pakai Chi Square ya pak?

**Reply by muhrisman** [reply]
Uji chi -square digunakan karena ingin membandingkan 2 grup atas signifikansi yang terlihat. Selain itu uji Chi Square digunakan karena output yang ingin dibandingkan berupa tabel kontingensi.
